.github/workflows/build-and-push-assets.yml

```diff
@@ -135,12 +135,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       NODE_OPTIONS: ${{ inputs.NODE_OPTIONS }}
```

Comment:
Is there a particular reason not to also move `NODE_OPTIONS` to the corresponding step(s)? This applies to the other workflows, too.

Reply:
I found nothing regarding `NODE_OPTIONS` in [`actions/setup-node`](https://github.com/actions/setup-node). This is something we need to talk about. As far as I remember, this is something we introduced because it is a global configuration for Node itself. So I guess we can move it to the `setup-node`-step as well (except we run somewhere else `node` as well?!). 👍🏻

Reply:
AFAIK, there's only one step per workflow where we run `node` commands, so it should be safe to move it there. Can you take care of it?

Reply:
Done here https://github.com/inpsyde/reusable-workflows/pull/180/commits/4020c5c75d609206a2bbde955b216b9552dbc440 👍🏻